release-deploy/action.yml

```diff
@@ -7,9 +7,20 @@ inputs:
     required: true
   version:
     required: true
+  slack_webhook:
+    required: true
 runs:
   using: "composite"
   steps:
+    - name: Slack Notification
+      uses: rtcamp/action-slack-notify@v2
+      env:
+          SLACK_CHANNEL: gyg
+          SLACK_COLOR: ${{ job.status }}
+          SLACK_TITLE: 'Deploying ${{ github.event.repository.name }} ${{ inputs.version }} to ${{ inputs.gyg_env }}_${{ inputs.gyg_region }}'
+          SLACK_MESSAGE: 'Please check ${{ github.event.repository.name }} for a new release after a few minutes.'
```

Comment:
Is this the right message? Perhaps just something like:
> Deployment successful.

Reply:
If the message is "Deployment successful.", it means the deployment is done. But we're deploying, which means the deployment is processing and the alert sends before starting to run the actual deployment.
![image](https://user-images.githubusercontent.com/23011040/170406364-4cc53d92-8cf7-4cd0-bee5-e4a588a99959.png)

Reply:
Ah, I must admit I assumed that this slack step was the very last thing that happened. Why don't we make it the last step so that we know it only runs if the deployment was actually successful?

Reply:
Oh, I think you want to alert someone about the deployment which is processing. Let me move to the last step if the deployment was actually successful.